python/torch_mlir_e2e_test/test_suite/conv.py

```diff
@@ -518,6 +518,34 @@ def forward(self, inputVec, weight):
 def _ConvolutionDeprecated2DCudnnModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(3, 3, 10, 10), tu.rand(3, 3, 2, 2))
 
+# ==============================================================================
+
+class Convolution2DKnownDimsGroups(torch.nn.Module):
```

Comment:
`KnownDims` is called `Static` in torch-mlir end2end tests
```suggestion
class Convolution2DGroupsStatic(torch.nn.Module):
```
and same below